env/dhis.env.example

```diff
@@ -0,0 +1,8 @@
+PALAU_DHIS_CLIENT_SECRET=
+PALAU_DHIS_PASSWORD=
+PALAU_DHIS_API_URL=
+DHIS_CLIENT_ID=
+DHIS_CLIENT_SECRET=
+DHIS_PASSWORD=
+DHIS_USERNAME=
+IS_PRODUCTION_ENVIRONMENT=
```

Comment:
`IS_PRODUCTION_ENVIRONMENT` should probably be in `servers.env` I think

Reply:
![Screen Shot 2024-04-04 at 1 45 14 PM](https://github.com/beyondessential/tupaia/assets/129009580/8b3e7312-e579-4e0f-ac5d-5121a96a9758)

I mainly added in the dhis file because it seems like, from the dhis-api readme that it is specific to dhis functionality?